ROADMAP.md

```diff
@@ -1,5 +1,5 @@
-Roadmap
-=======
+ Roadmap
+========
```

Comment:
In this case, it seems like it would be better to be consistent with the `README.md` and use the `#` style for headings.